neuro_san/registries/manifest.hocon

```diff
@@ -22,6 +22,9 @@
     "music_nerd_pro.hocon": true,
     "gist.hocon": true,
 
+    # This one is an example of agnets calling tools that has no parameters.
+    "date_time.hocon": true,
```

Comment:
Might be worthwhile adding a unit test hocon file to the mix to be sure that no-params tools continue to work.